crates/fm/Cargo.toml

```diff
@@ -8,6 +8,14 @@ edition = "2018"
 
 [dependencies]
 codespan-reporting = "0.9.5"
+wasm-bindgen = { version = "*", features = [
```

Comment:
In order to avoid needing wams-bindgen here, we could have the file manager take a trait instead like  FileManager<FileReader>

Reply:
Issue opened for this: https://github.com/noir-lang/noir/issues/322